Examples/Algorithms/Geant4/include/ActsExamples/Geant4/SensitiveSteppingAction.hpp

```diff
@@ -31,6 +31,10 @@ class SensitiveSteppingAction : public G4UserSteppingAction {
     bool neutral = false;
     bool primary = true;
     bool secondary = false;
+
+    // When to kill a track
+    double maxAbsZ = std::numeric_limits<double>::infinity();
+    double maxR = std::numeric_limits<double>::infinity();
```

Comment:
If you have a dedicated `ParticleKillAction` - why does the stepping action need to know about this?
Isn't the idea of actions that they can be done w/o communication with another?